projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -3237,8 +3237,186 @@ public string FatherEthnicityLiteral
+        }
+
+        /// <summary>Mother's Education Level Edit Flag Helper</summary>
+        /// <value>Mother's Education Level Edit Flag.</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.MotherEducationLevelEditFlag = VRDR.ValueSets.EditBypass01234.EditPassed;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Mother's Education Level Edit Flag: {ExampleBirthRecord.EducationLevelHelperEditFlag}");</para>
+        /// </example>
+        [Property("Education Level Edit Flag Helper", Property.Types.String, "Decedent Demographics", "Mother's Education Level Edit Flag Helper.", false, VR.IGURL.EducationLevel, false, 34)]
+        [PropertyParam("code", "The code used to describe this concept.")]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='57712-2')", "")]
+        public string MotherEducationLevelEditFlagHelper
+        {
+            get => GetObservationValueHelper();
+            set => SetObservationValueHelper(value, VR.ValueSets.EditBypass01234.Codes);
+        }
+
+        /// <summary>Father's Education Level.</summary>
+        /// <value>the father's education level. A Dictionary representing a code, containing the following key/value pairs:
+        /// <para>"code" - the code</para>
+        /// <para>"system" - the code system this code belongs to</para>
+        /// <para>"display" - a human readable meaning of the code</para>
+        /// </value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>Dictionary&lt;string, string&gt; elevel = new Dictionary&lt;string, string&gt;();</para>
+        /// <para>elevel.Add("code", "BA");</para>
+        /// <para>elevel.Add("system", VR.CodeSystems.EducationLevel);</para>
+        /// <para>elevel.Add("display", "Bachelor’s Degree");</para>
+        /// <para>ExampleBirthRecord.FatherEducationLevel = elevel;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Father's Education Level: {ExampleBirthRecord.FatherEducationLevel["display"]}");</para>
+        /// </example>
+        [Property("Father's Education Level", Property.Types.Dictionary, "Education Profiles", "Father's Education Level.", true, VR.IGURL.EducationLevel, false, 78)]
+        [PropertyParam("code", "The code used to describe this concept.")]
+        [PropertyParam("system", "The relevant code system.")]
+        [PropertyParam("display", "The human readable version of this code.")]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='87300-0').value.coding", "")]
+        public Dictionary<string, string> FatherEducationLevel
+        {
+            get => GetObservationValue("87300-0");
+            set => SetObservationValue(value, "87300-0", CodeSystems.LOINC, "Highest level of education Father", VR.ProfileURL.EducationLevel, NEWBORN_INFORMATION_SECTION);
```

Comment:
See earlier comment re the appropriate section, I think this should be FATHER_INFORMATION_SECTION.